FV3/gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -184,6 +184,7 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
     real (kind=kind_phys), pointer :: smc (:,:)   => null()  !< soil moisture content
     real (kind=kind_phys), pointer :: stc (:,:)   => null()  !< soil temperature content
     real (kind=kind_phys), pointer :: slc (:,:)   => null()  !< soil liquid water content
+    real (kind=kind_phys), pointer :: atm_ts (:,:) => null() !< surface temperature from dynamical core
```

Comment:
This is a nice name change; indeed `sst` makes it a little unclear where it's coming from in the dynamical core.